crates/transaction-emitter-lib/src/emitter/transaction_executor.rs

```diff
@@ -25,48 +26,94 @@ impl RestApiTransactionExecutor {
         self.rest_clients.choose(&mut rng).unwrap()
     }
 
-    async fn submit_and_check(
```

Comment:
if this is matching wrong functions, disable whitespace differences, i.e. :

https://github.com/aptos-labs/aptos-core/pull/6513/files?diff=split&w=1

and it will match nicely